app/components/web-components/vl-modal-verify.js

```diff
@@ -28,7 +28,7 @@ export default class WebComponentsVlModalVerify extends Component {
   }
 
   get verifyButtonText() {
-    return this.intl.t(this.args.buttonText ?? 'delete');
+    return this.args.verifyButtonText ?? this.intl.t('delete');
   }
```

Comment:
why this here?
Seems like a commit for a different PR

Reply:
Found a hardcoded dutch string because of this.
`@verifyButtonText="Verwijderen"` in `subcase-header.hbs` line 86
Removing it in favor of default translation of 'delete'

Reply:
fixed it

Reply:
This was a mistake from the Octane refactor PR. Noticed it here(because the modals for publication in Kort Bestek had a blue "Verwijderen" button) and decided to fix it here because it was such a small change to begin with.